tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -36,7 +36,7 @@ private[scalanative] object LLVM {
       incCompilationContext: IncCompilationContext
   ): Seq[Path] = {
     // generate .o files for all included source files in parallel
-    paths.par.map { path =>
```

Comment:
TODO: restore this, as I was trying to not have interleaved output from so many failing compilations.

A better approach is to just grab one of the failing `.ll` files and run it through either `llc` or `clang -c <file.ll>` - it's enough to reproduce the errors.